addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -113,11 +113,6 @@ const ColorPaletteWidget = Widget.extend({
             });
         }
 
-        // Render custom colors
```

Comment:
Commit message:
- "fixer le thème aux couleurs personnalisées", ca ne veut rien dire
- Pas de point à la fin du titre
- "set to" instead of "set from", no?
- "it's name"
- "Also before the theme tab the selected color if it was custom would always appear in the palette and be marked selected" -> relu 14x et je sais pas il manque un verbe ou un sujet, je sais pas du tout :')